docs/index.rst

```diff
@@ -48,6 +48,7 @@ Project Info
     step_generic
     step/hla_typing
     step/igv_session_generation
+    step/mobile_element_insertion
```

Comment:
Should probably be called targeted_seq_mei_calling as we have wgs_mei_calling already.